mssqlcli/mssqlcliclient.py

```diff
@@ -20,7 +19,8 @@ def generate_owner_uri():
     return u'mssql-cli-' + uuid.uuid4().urn
 
 
-class MssqlCliClient(object):
+class MssqlCliClient:
+    # pylint: disable=too-many-instance-attributes
```

Comment:
Just a consideration: what pylint rules are we using? Are they mandated by a standard Microsoft process? I'm wondering whether for some classes of issue, relaxing the rules is better than fixing / spot-ignoring them in each place.

Reply:
We're using pylint's default configuration. I don't know of anything that's Microsoft mandated.

It is possible to relax the rules, however, I generally like the pylint suggestions moving forward. Even with this specific commented line, where I didn't feel like refactoring the class for the sake of this issue, I think that moving forward if we adhere to these rules, the better our code will be.

With that being said, it's definitely worth having a discussion with Brian (when he's back) about what rules we'd like to ignore moving forward.